resources/ory/charts/oathkeeper/values.yaml

```diff
@@ -130,9 +130,8 @@ oathkeeper:
 
 secret:
   # -- switch to false to prevent creating the secret
-  enabled: true
+  enabled: false
   # -- Provide custom name of existing secret, or custom name of secret to be created
-  nameOverride: ""
```

Comment:
comments related to `nameOverride` can also be removed

Reply:
underneath is also commented nameOverride
`# nameOverride: "myCustomSecret"`
so i think we can keep them